src/components/views/rooms/EventTile.tsx

```diff
@@ -976,12 +1054,16 @@ export default class EventTile extends React.Component<IProps, IState> {
             onFocusChange={this.onActionBarFocusChange}
         /> : undefined;
 
-        const showTimestamp = this.props.mxEvent.getTs()
-            && (this.props.alwaysShowTimestamps
-            || this.props.last
-            || this.state.hover
-            || this.state.actionBarFocused);
-
+        const showTimestamp = (
+            this.props.mxEvent.getTs() &&
+            (
+                this.props.alwaysShowTimestamps ||
+                this.props.last ||
+                this.state.hover ||
+                this.state.actionBarFocused ||
+                this.state.contextMenu
+            )
+        );
```

Comment:
There's enough conditions and types in here that `!!` on the non-bools might add subconscious clarity, but not required.